test/statementTest.js

```diff
@@ -168,6 +168,44 @@ describe('Statement Class Tests', () => {
     });
   });
 
+  describe('bindParameters', () => {
+    before('create table for test', async () => {
+      const pool = new DBPool({ url: '*LOCAL' }, { incrementSize: 2 });
+      const createTable = `CREATE TABLE ${schema}.SCORES(team VARCHAR(100) ALLOCATE(20), score INTEGER)`;
```

Comment:
In [#78](https://github.com/IBM/nodejs-idb-pconnector/pull/78/files#diff-860887daeecc0a57c5a03dcca9925ff1e1ffde296709a5600c2cba3c09a27c52R31) The test scores table is added. If that gets merged first no need to duplicate creating it here.

Reply:
That was merged. There are now conflicts which can be cleaned up and I'm guessing this can then be removed?

Reply:
Yes This can just be cleaned up by removing the `afterEach` hook.